apps/code-examples/src/app/code-examples/modals/inline-help/modal-demo.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { SkyModalConfiguration, SkyModalService } from '@skyux/modals';
+
+import { ModalDemoModalComponent } from './modal-demo-modal.component';
+
+@Component({
+  selector: 'app-modal-demo',
+  templateUrl: './modal-demo.component.html',
+})
+export class ModalDemoComponent {
+  public modalSize = 'medium';
+
+  #modalSvc: SkyModalService;
+
+  constructor(modalSvc: SkyModalService) {
+    this.#modalSvc = modalSvc;
+  }
+
+  public onOpenModalClick(): void {
+    const modalInstanceType: any = ModalDemoModalComponent;
```

Comment:
Instead of using `any` can we just pass the `ModalDemoModalComponent` type directly into the `open` method below?